lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py

```diff
@@ -187,6 +205,71 @@ def exec_module(self, **kwargs):
         else:
             self.results['ansible_facts']['azure_vmss'] = self.list_items()
 
+        if self.format == 'curated':
+            for index in range(len(self.results['ansible_facts']['azure_vmss'])):
+                vmss = self.results['ansible_facts']['azure_vmss'][index]
+                subnet_name = None
+                load_balancer_name = None
+                virtual_network_name = None
+                ssh_password_enabled = False
+
+                try:
+                    subnet_id = (vmss['properties']['virtualMachineProfile']['networkProfile']['networkInterfaceConfigurations'][0]
+                                 ['properties']['ipConfigurations'][0]['properties']['subnet']['id'])
+                except:
+                    pass
```

Comment:
Would be to not swallow the error but at least log it out somewhere for future reference